sound/soc/sof/ipc4-topology.c

```diff
@@ -2020,6 +2194,15 @@ static enum sof_tokens src_token_list[] = {
 	SOF_COMP_EXT_TOKENS,
 };
 
+static enum sof_tokens process_token_list[] = {
+	SOF_COMP_TOKENS,
+	SOF_AUDIO_FMT_NUM_TOKENS,
+	SOF_IN_AUDIO_FORMAT_TOKENS,
+	SOF_OUT_AUDIO_FORMAT_TOKENS,
+	SOF_AUDIO_FORMAT_BUFFER_SIZE_TOKENS,
+	SOF_COMP_EXT_TOKENS,
```

Comment:
why is your new token for OUTPUT_FMT_CNT not a part of this list? If it is not needed, why introduce it at all?

Reply:
It's in SOF_COMP_EXT_TOKENS